dockers/docker-database/database_config.json.j2

```diff
@@ -5,6 +5,12 @@
             "port" : 6379,
             "unix_socket_path" : "/var/run/redis{{NAMESPACE_ID}}/redis.sock",
             "persistence_for_warm_boot" : "yes"
+        },
+        "redis_global":{
+            "hostname" : "redis_global.server",
+            "port": 6385,
+            "unix_socket_path": "/var/run/redis-global/redis_global.sock",
```

Comment:
This is not used. Maybe get rid of it?

Reply:
unic_soucket_path has to be valid when start redis server.

Reply:
sonic-db-cl -n asic0 PING will use domain socket to ping instead of TCP, we have to make it valid so that PING won't fail in multi-asic env.